### PR TITLE
Create dedup.sql

### PR DESCRIPTION
**Pull request by elyse-weiss**

### CODE REVIEW

File: macros/sql/dedup.sql [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{% macro coalesce_to_zero(field) -%}
+
+    coalesce( {{ field }} ,0) {{field}}
+
+{%- endmacro %}
+
+
+{% macro normalize_resource(entity_name, resource_number) %}
```
Comment (jthandy) [inline macros/sql/dedup.sql]: pretty sure this macro doesn't belong in this file (or PR)!

File: macros/sql/dedup.sql [hunk 2]
```diff
@@ -0,0 +1,58 @@
+{% macro coalesce_to_zero(field) -%}
```
Comment (jthandy) [inline macros/sql/dedup.sql]: pretty sure this macro doesn't belong in this file (or PR)!

**Comment by drewbanin** [comment]
We discussed a different implementation of this macro that would dedupe using an `md5` over a number of rows. Closing

**Comment by blake-enyart** [comment]
@drewbanin I was just looking for deduplication functions in `dbt-utils` and came across this PR. Which function in `dbt-utils` resolves this PR?

**Comment by drewbanin** [comment]
hey @blake-enyart - I don't believe we ever got this implemented! If you think it could be useful, feel free to open a new issue and we can discuss some approaches over there! Thanks :)

**Pull request closed without merging**